docker/srv/salt/top.sls

```diff
@@ -1,3 +1,5 @@
 base:
   '*':
     - vim
+    - python
+    - users
```

Comment:
why was this introduced?

Reply:
Wow. Sorry about this. It's my local change to test highstate output. I will fix this. Good catch :)